contents/teams/replay/objectives.mdx

```diff
@@ -1,24 +1,18 @@
-### 🎯 Goal 1: Quality and Consolidation for Session Replay (web & mobile)
+### 🎯 Goal 1: People think of PostHog as a mobile solution
 
-Why? 
+* Mobile replay for React Native
+* Lots of content on how to use PostHog on Mobile
 
-We've done a lot of investigation and preparation work and we want to capitalize on that
+### 🎯 Goal 2: Error tracking in people's hands
 
-* Mobile replay out of beta for Android/iOS
-* Make capture and playback as complete, performant and easy as possible
-* Build the best UX for finding and playing recordings
+* the top 5 features implemented
```

Comment:
I think we should be more specific here, what are the top 5 features?

Reply:
For me they are:
- Grouping
- Stack traces
- SDK support
- Alerting
- Integrated with other products (this one is less important than the other 4 for me)

Reply:
fine with listing these here (although going to be duplicated with mega issue)
only concern is these are a little bit "techno bits" rather than "user needs" but 🤷

Reply:
We can also rewrite in order of priority with a more user-centric language such as:

- SDK captures errors
- Stack traces are actionable and human-readable
- Similar errors are grouped together
- New errors with high volume trigger the Alerting system
- Integrated with other products such as Session replay